microsoft-edge/webview2/code-samples-links.md

```diff
@@ -1,213 +1,49 @@
-|---|---|---|
-| WinUI 2 (UWP) browser | [webview2_sample_uwp](https://github.com/MicrosoftEdge/WebView2Samples/blob/main/SampleApps/webview2_sample_uwp#readme) | Embeds a WebView2 control within a UWP application.<br/><br/>  Built as a UWP Visual Studio 2019 project.  Uses C++ and HTML/CSS/JavaScript in the WebView2 environment. |
-| Win32 C++ | [WebView2APISample](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/WebView2APISample#readme) | Embeds a WebView2 control within a Win32 native application.  Illustrates a selection of WebView2 event handlers and API methods that allow a native Win32 application to directly interact with a WebView2 control and vice versa.<br/><br/>  Built as a Win32 project in [Visual Studio 2019](https://visualstudio.microsoft.com/vs/).  Uses C++ and HTML/CSS/JavaScript in the WebView2 environment.<br/><br/>The `.sln` file is in the parent, [SampleApps](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps) directory. |
-| Win32 C++ with Visual Composition | [WebView2SampleWinComp](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/WebView2SampleWinComp) | Embeds a WebView2 control within a Win32 native application.  Uses [Windows Runtime Composition APIs](/uwp/api/windows.ui.composition), also called the Visual layer, to take advantage of the latest Windows 10 or later UI features and create better look, feel, and functionality in C++ Win32 applications.<br/><br/>  Built as a Win32 project in Visual Studio 2019.  Uses C++ and HTML/CSS/JavaScript in the WebView2 environment. |
-| WinForms | [WebView2WindowsFormsBrowser](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/WebView2WindowsFormsBrowser) | Embeds a WebView2 control within a Windows Forms application.<br/><br/>  Built as a Windows Forms project in Visual Studio 2019.  Uses C# and HTML/CSS/JavaScript in the WebView2 environment. |
-| Windows Presentation Foundation (WPF) .NET | [WebView2WpfBrowser](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/WebView2WpfBrowser) | Embeds a WebView2 control within a WPF application.<br/><br/>  Built as a WPF project in Visual Studio 2019.  Uses C# and HTML/CSS/JavaScript in the WebView2 environment. |
-| Chrome DevTools Protocol (CDP) in WPF | [WV2CDPExtensionWPFSample](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/WV2CDPExtensionWPFSample) | Uses Chrome [DevTools Protocol](../devtools-protocol-chromium/index.md) functions using a `DevToolsProtocolHelper` object in WebView2.  Demonstrates the usage patterns of the WebView2 CDP extension in WPF.  This application is built with the [WebView2 CDP Extension](https://aka.ms/webviewcdp) that defines all CDP methods, events, and types.<br/><br/>  Built as a WPF project in Visual Studio 2019.  Uses C# in the WebView2 environment. |
-
-#### Deployment samples
+#### Articles, readmes, sample directories, and solution files
```

Comment:
What is this section trying to convey?

Reply:
I think this table is dup'ing the TOC and is dup'ing details that are already in the individual sample articles.  There is one point that does belong in this overarching article: the special nature of the Win32 / multi-platform sample/solution (currently called out in Notes column on right side of table).  For maintainability, and simplicity of presentation, we need to eliminate dup'ing of details.  Not sure if this applies to the getstart landing page's table & Deployment landing page's table.

Consider the Readme's Redesign PR in the Samples repo, which simplifies & removes dup'n of details; that's working, and can do same simplifying/streamlining here.

Reply:
removed